dom.bs

```diff
@@ -2668,6 +2681,11 @@ indicated in the <a for=/>remove</a> algorithm below.
   <p>If <var>node</var> has an <a>inclusive descendant</a> that is a <a>slot</a>, then:
 
   <ol>
+   <li><p>for each <a>slot</a> <var>slot</var> in <var>node</var>'s <a for=tree>inclusive descendants</a>,
+   if <var>slot</var>'s <a for=tree>root</a> is a <a for=/>shadow root</a> whose <a for=tree>root</a>'s
+   <a for=ShadowRoot>slot assignment</a> is "<code>manual</code>", then set <var>slot</var>'s
+   <a>manually assigned nodes</a> to an empty set.
```

Comment:
Hm... this would be different from what we've been discussing in AOM:
https://github.com/whatwg/html/issues/4925

Reply:
How does this relate to that?

Reply:
Well, it would mean that that this slot.assingedNodes would behave differently from element ID reflection, which seems bad.